uoyweek.cpp

```diff
@@ -30,47 +46,49 @@ class Term {
 
     string toString() {
         char yearStr[128];
-        //Assign result of asctime to string due to  weird memory management of asctime
+        /* Assign result of asctime to string due to  weird memory management of
+         * asctime */
         char start_string[128];
         strncpy(start_string, asctime(&start), 128);
         char end_string[128];
         strncpy(end_string, asctime(&end), 128);
-        //sprintf is fine for actual values but results of asctime, etc. should be put in string in memory first
-        sprintf(yearStr, "%s (%d-%d) %s %s", code.c_str(), ((&start)->tm_year+1900), ((&end)->tm_year+1900), start_string, end_string);
+        /* sprintf is fine for actual values but results of asctime, etc. should
+         * be put in string in memory first */
+        sprintf(yearStr, "%s (%d-%d) %s %s",
+                code.c_str(),
+                (&start)->tm_year + 1900,
+                (&end)->tm_year + 1900,
+                start_string,
+                end_string);
         return yearStr;
     }
 };
 
-Term * aut  = new Term("aut", (time_t) 1506297600l, (time_t) 1512086400l);
-Term * spr  = new Term("spr", (time_t) 1515369600l, (time_t) 1521158400l);
-string days[] = {
-    "sunday",
-    "monday",
-    "tuesday",
-    "wednesday",
-    "thursday",
-    "friday",
-    "saturday"
-};
+const int AUT_START = 1506297600l;
+const int AUT_END = 1512086400l;
```

Comment:
alignment

Reply:
This is prettified temp code, should be gone soon